stream/src/main/mima-filters/1.0.x.backwards.excludes/pr-1619-remove-supervised-group-stage-logic.backwards.excludes

```diff
@@ -0,0 +1 @@
+ ProblemFilters.exclude[MissingClassProblem]("org.apache.pekko.stream.impl.fusing.SupervisedGraphStageLogic")
```

Comment:
we can't do this in Pekko 1.1 - this will need to be in Pekko 1.2 or even 2.0

Reply:
Anything in the `impl` package is internal implementation and not public API and the class is `private[pekko]`, so it might be fine?

Reply:
@He-Pin can you add a license header (the standard Apache one, not the one that mentions Akka)? Can you add a new line at the end of the file?

Reply:
I missed that:(